cserver.py

```diff
@@ -293,8 +293,9 @@ def query():
                         res.append(i.value)
             print(len(res))
     elif string == 'get_user_ratings':
-        msin = request.args.get('msin')
+      """DEPRECATED SINCE PYTHON-REIN COMMIT 'Use local database for ratings data...'"""
```

Comment:
The spaces on this and the next line throw an expected-indent error for me. I just removed the empty line and shifted the """ line over one to fix.